src/regression_tracker.py

```diff
@@ -72,6 +72,7 @@ def _create_regression(self, failed_node, last_pass_node):
         """Method to create a regression"""
         regression = {}
         regression['kind'] = 'regression'
+        regression['result'] = 'fail'
```

Comment:
As we are always creating regression nodes with `result=fail`, how about setting the field by default to `fail`?